githubdelivery/src/main/java/io/dockstore/githubdelivery/GithubDeliveryS3Client.java

```diff
@@ -124,29 +124,7 @@ private String getObject(String key) throws IOException {
         ResponseInputStream<GetObjectResponse> object = s3Client.getObject(objectRequest);
         return IOUtils.toString(object, StandardCharsets.UTF_8);
     }
-    private PushPayload getGitHubPushPayloadByKey(String eventType, String body, String key) throws IOException, NoSuchKeyException {
```

Comment:
Moved this to GitHubDeliveryHelper so I could eliminate some duplication and add a (crude) unit test.